README.md

```diff
@@ -182,3 +182,45 @@ mysql> exit;
 > mysql -u root -p
 > password
  ```
+
+Create a Database `ecommercedb` for this application using the following command:
+```sh
+CREATE DATABASE ecommercedb;
+```
+
+Since we will be using the database, run the following command
+```sh
+USE ecommercedb;
+```
+
+## Table Creation:
+
+Use the following command to create the tables :
+
+> Categories
+
+```
+CREATE TABLE `Categories` (
```

Comment:
This is erronous - ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 7

Reply:
Resolved in next commit - 7b6b666eb0b5fdc774c8debc28c4ee57f3bdda6c